vms/platformvm/txs/mempool/mempool.go

```diff
@@ -69,16 +69,15 @@ type Mempool interface {
 	// It's guaranteed that the returned tx, if not nil, is a StakerTx.
 	PeekStakerTx() *txs.Tx
 
-	// Note: dropped txs are added to droppedTxIDs but not
-	// not evicted from unissued decision/staker txs.
-	// This allows previously dropped txs to be possibly
-	// reissued.
+	// Note: dropped txs are added to droppedTxIDs but not evicted yet
```

Comment:
I don't think we should add `yet` here, that seems to imply they are evicted... which isn't the case. Specifically the flow that is happening here is:

txs are marked as dropped when they fail verification. A tx may be verified against multiple different states. So, just because it failed verification, doesn't mean it can never be accepted. A tx is allowed in to the mempool if it is valid on top of the currently preferred state. However, it may have been previously verified against a different state at which it was invalid (marking it as dropped there). 

Essentially, this method is only used for explaining what happened to a transaction to the API caller... It is not intended to be used for preventing the issuance of invalid transactions.